benchmarks/data/model1_hmf.txt

```diff
@@ -0,0 +1,14 @@
+#[0]log10[M/Msun] [1]sigma [2]log10[1/sigma] [3]log10[M^2/rho_m*dn/dM]
```

Comment:
where did this file come from? Is it the same one being used in C?

Reply:
I copied the C file. I actually could not find the code that generated this benchmark.

Reply:
No, that's fine. Sounds good.

Reply:
I just didn't remember the original file having log(1/sigma)